crates/ruff/resources/test/fixtures/flake8_pyi/PYI041.pyi

```diff
@@ -0,0 +1,38 @@
+from typing import (
+    Union,
+)
+
+from typing_extensions import (
+    TypeAlias,
+)
+
+TA0: TypeAlias = int
+TA1: TypeAlias = int | float | bool
+TA2: TypeAlias = Union[int, float, bool]
```

Comment:
I think we should document the limitation that this rule won't check for type aliases.

Reply:
Added a comment. Thanks!